liberty-template/template.yaml

```diff
@@ -41,20 +41,20 @@ spec:
         cluster_id:
           title: Cluster Id
           type: string
-          description: Id of the cluster
-          default: <XXX_XXX_XXX>
+          description: Id of the cluster (Everything in the host after .apps)
```

Comment:
I thought the ID included the `.apps` part. I may be remembering that incorrectly. If it does, then I wonder if people will interpret "Everything in the host after .apps" as everything **_after and not including_** the `.apps` part.

Reply:
I've always included the .apps part... I'm wondering if this field even makes a difference?